src/components/notification/notification.scss

```
@@ -138,26 +171,15 @@
     grid-template-columns: auto 1fr auto;
     align-items: flex-start;
   }
+
+  :host([size='s']) & {
+    @include sbb.text-xs--regular;
+  }
 }
 
 .sbb-notification__icon {
   color: var(--sbb-notification-icon-color);
-
-  @include sbb.mq($from: small) {
-    margin-block-start: calc(
-      ((1em * var(--sbb-typo-line-height-body-text)) - var(--sbb-size-icon-ui-small)) / 2
-    );
-
-    :host(:where([data-slot-names~='title'], [title-content])) & {
-      margin-block-start: calc(
-        (
-            (var(--sbb-font-size-title-5) * var(--sbb-typo-line-height-body-text)) - var(
-                --sbb-size-icon-ui-small
-              )
-          ) / 2
-      );
-    }
-  }
+  margin-block-start: var(--sbb-notification-icon-margin-block-start, 'initial');
```

Does that not work? alternatively, just set it to 0 as standard?
```suggestion
  margin-block-start: var(--sbb-notification-icon-margin-block-start);
```

---

I would work without a default value too. But, by having one, I feel like it adds the explicit meaning "it is meant to be undefined in some cases".
If you're ok, I'll change it to '0'

---

ok. Alternatively, in this case, we normally define it directly in the host block. But both is fine :)

---

I'd prefer that too. I did not do it that way because it's slightly redundant and in general, we try to be very attentive to efficiency.
But I believe I'm overthinking as usual :|